charger/config.go

```diff
@@ -27,6 +27,8 @@ func NewFromConfig(log *api.Logger, typ string, other map[string]interface{}) ap
 		c = NewEVSEWifiFromConfig(log, other)
 	case "simpleevse", "evse":
 		c = NewSimpleEVSEFromConfig(log, other)
+	case "porsche", "audi", "bentley", "mcc":
+		c = NewMCCFromConfig(log, other)
```

Comment:
Ich würde das MCC ausschreiben für Wiedererkennung- lange Namen stören hier nicht unbedingt. Vielleicht als "MobileConnect"?

Reply:
...von aussen sieht das dann nach idiomatischem go aus: `charger.NewMobileConnectFromConfig`

Reply:
Hatte ich Anfangs genau so, aber die anderen sind so kurz, da dachte ich ... :)

Also mache ich.

Reply:
Würdest du dann auch jedes Vorkommen "MCC" durch "MobileConnect" ersetzen? Fände ich dann "schöner"

Reply:
Nicht in der methodensignatur bei *mcc. Da würde ich sogar *c nehmen wie alle anderen charger auch. Aber egal, *mcc geht auch.